code/__DEFINES/~skyrat_defines/projectiles.dm

```diff
@@ -4,6 +4,7 @@
 #define CALIBER_473MM "473mm"
 #define CALIBER_6MM "6mm"
 #define CALIBER_10MMAUTO "10x25mm"
+#define CALIBER_9MMPEACE "9x19mm"
```

Comment:
Name them 9mm Peacekeeper / Peacekeeper 9mm / just "Peacekeeper" rounds, we're trying to reduce the number of gunnut caliber names.

Reply:
Its used with defines so coders see CALIBER_9MMPEACE 
Until it gets a flavor change this is good enough